2.6/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -10,7 +10,11 @@ set -o pipefail
 export MONGODB_DATADIR=/var/lib/mongodb/data
 export CONTAINER_PORT=27017
 # Configuration settings.
-export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
+if [[ ${MONGODB_NOPREALLOC:-true} == "true" ]]; then
+  export MONGODB_PREALLOC="false"
+else
+  export MONGODB_PREALLOC="true"
+fi
```

Comment:
you should probably surround this block w/ a check to see if MONGODB_PREALLOC is set and if so, respect that value directly.

And then update the image doc to deprecate MONGODB_NOPREALLOC in preference to MONGODB_PREALLOC